meta/FlexibleSUSY.m

```diff
@@ -2505,17 +2502,56 @@ corresponding tadpole is real or imaginary (only in models with CP
                CXXDiagrams`IndexDiagramFromGraph[diagrams[[i,j]], graphs[[i]]],
                   graphs[[i]]
                 ], 16] <> " * " <>
-                ToString @ AMuon`CXXEvaluatorForDiagramFromGraph[diagrams[[i,j]], graphs[[i]]] <>
+                ToString @ AMM`CXXEvaluatorForDiagramFromGraph[diagrams[[i,j]], graphs[[i]]] <>
                 "::value({" <> muonIndex <> "}, context, qedqcd);\n"
          ];
       ];
 
+      leptonPhysicalMass =
+         If[GetParticleFromDescription["Leptons"] =!= Null,
+"template <typename Lepton>
+double leptonPhysicalMass(const softsusy::QedQcd& qedqcd, int idx)
+{
+   double lepton_physical_mass;
+   switch(idx) {
+      case 0: lepton_physical_mass = qedqcd.displayPoleMel(); break;
+      case 1: lepton_physical_mass = qedqcd.displayPoleMmuon(); break;
+      case 2: lepton_physical_mass = qedqcd.displayPoleMtau(); break;
+   }
```

Comment:
Can you add a `default:` case which throws an exception if the index is out of bounds?

Reply:
Alternatively we could also create a new convenience member function in `QedQcd` class which does exactly what is done here, i.e. access a fermion pole mass via an index.

Reply:
Partially done in 34b6d780612a99b8ed268e8c878ba2201ed25ffc. I don't have a preference either way. For now I've added a `throw OutOfBoundsError` that would print something like
```
Cannot compute anomalous magnetic moment of `Fe(4)`
```
but this is not correctly catched in `_observables` so it prints
```
Block OBSINFO
     0     3   all observables: unknown exception thrown
```
Should I extend it?

Reply:
Extending it would be better, I think.